blocks/persistence/memory.h

```diff
@@ -166,6 +166,28 @@ class MemoryPersister {
     return idxts_t(index, timestamp);
   }
 
+  template <current::locks::MutexLockStatus MLS>
+  idxts_t PersisterPublishUnsafeImpl(const std::string& entry_json) {
+    current::locks::SmartMutexLockGuard<MLS> lock(container_->memory_persister_container_mutex_);
+    const auto head = container_->head_;
+    const auto tab_stop = entry_json.find('\t');
```

Comment:
+1 tab_pos ("tab stop" is the number of column in a tab-separated line)

и моя школа называть по-разному (контекстно-независимо) более-менее глобальные, публичные объекты, например, типы, методы, поля; 

локальные переменные моя школа называть согласно тому, что в них лежит (плюс иногда суффиксом тип того, что лежит, когда в одной области видимости находится несколько переменных про что-то одно); 

и если в разных местах кода в переменных или полях лежит одно и то же, то стараться называть их одинаково (с одинаковым префиксом), чтобы при глобальном рефакторинге найти все связанные друг с другом места.

Reply:
Guys, don't worry, actually I don't know where I got this `tab_stop` from, because everywhere else I used `tab_pos`. Probably, it's some strange kind of typo :)